curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a240c67f3dbb1a1e6d95ee.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-10
 
 # --description--
 
-Give your `#game` a maximum width of `500px` and a maximum height of `400px`. Set the `background-color` to `lightgray` and the `color` to `white`. Finally, use margins to center it and give it `10px` of padding on all four sides.
+Give your `#game` a maximum width of `500px` and a maximum height of `400px`. Set the `background-color` to `lightgray` and the `color` to `white`. Use margins to center it by setting the top and bottom margin to `0`, and the left and right margin to `auto`. Finally, give it `10px` of padding on all four sides.
```

Comment:
I am kind on the fence with this change because we do have a hint that tells users the correct answer if they get the `margin: 0 auto;` wrong. 

So, I am wondering if we should leave it and the hint would suffice 🤔

Reply:
I think it'd be clearer to learners that way. So they won't need to test the code before seeing they could have used `margin: 0 auto`.

That's what the feedback says. 

What do you think?

Reply:
I could go either way, though I lean towards being more explicit in this case, especially since it's the first project for the JS cert and a refresher for HTML and CSS.

This change looks good to me. Hopefully between the clearer instructions here and the hint there won't be much confusion on how we expect the element to be centered.

Reply:
Okay. I left it like that.